application/libraries/Save.php

```diff
@@ -211,11 +211,11 @@ function saveSurvey()
                 $mailer->emailType = 'savesurveydetails';
                 $mailer->isHTML(false);
                 $mailer->Subject = gT("Saved Survey Details") . " - " . $thissurvey['name'];
-                $message  = gT("Thank you for saving your survey in progress.  The following details can be used to return to this survey and continue where you left off.  Please make sure to remember your password - we cannot retrieve it for you.");
+                $message  = gT("Thank you for saving your survey in progress.  The following details can be used to return to this survey and continue where you left off.  Please make sure to remember your password - we cannot retrieve it for you.", "unescaped");
                 $message .= "\n\n" . $thissurvey['name'] . "\n\n";
                 $message .= gT("Name") . ": " . Yii::app()->getRequest()->getPost('savename') . "\n";
```

Comment:
Please apply this fix also for the other translated textst in here (Name,Password, etc)-